web/locale/zh_CN/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,886 @@
+msgid "Show only followed users activity."
+msgstr "仅显示我关注的用户动态。"
+
+#: templates/newsfeed.html:18
+msgid "Show all users activity."
+msgstr "显示所有用户动态。"
+
+#: templates/newsfeed.html:35
+msgid "Newsfeed seems empty."
+msgstr "动态为空。"
+
+#: templates/notifications.html:8
+msgid "Notifications"
+msgstr "提醒"
+
+#: templates/stats.html:7 templates/tabs.html:10
+msgid "Statistics"
+msgstr "统计"
+
+#: templates/stats.html:10
+msgid "most active users"
+msgstr "最活跃的用户"
+
+#: templates/stats.html:11
+msgid "most supported users"
+msgstr "被支持最多的用户"
+
+#: templates/stats.html:13
+msgid "most crowded arguments"
+msgstr "争议最多的论证"
+
+#: templates/stats.html:14
+msgid "most supported premises"
+msgstr "被支持最多的前提"
+
+#: templates/stats.html:15
+msgid "most fallacy premises"
+msgstr "最荒谬的前提"
+
+#: templates/stats.html:20
+msgid "this week"
+msgstr "本周"
+
+#: templates/stats.html:21
+msgid "this month"
+msgstr "本月"
+
+#: templates/stats.html:22
+msgid "all time"
+msgstr "所有的时间"
+
+#: templates/tabs.html:4
+msgid "Newsfeed"
+msgstr "信息流"
+
+#: templates/tabs.html:6
+msgid "Featured"
+msgstr "精选"
+
+#: templates/tabs.html:8
+msgid "Recently Updated"
+msgstr "最新提交"
+
+#: templates/tabs.html:12
+msgid "Search"
+msgstr "搜索"
+
+#: templates/auth/login.html:7 templates/auth/login.html.py:11
+#: templates/auth/password_reset_complete.html:9
+msgid "Login"
+msgstr "登录"
+
+#: templates/auth/login.html:7 templates/auth/register.html:7
+msgid "or login with twitter"
+msgstr "或使用 twitter 登录"
+
+#: templates/auth/login.html:14
+msgid "Forgot your password?"
+msgstr "忘记密码了?"
+
+#: templates/auth/password_reset_complete.html:8
+msgid "The password changed successfully."
+msgstr "密码修改成功。"
+
+#: templates/auth/password_reset_confirm.html:7
+msgid "Enter your password"
+msgstr "输入密码"
+
+#: templates/auth/password_reset_confirm.html:17
+msgid "New password (again)"
+msgstr "新的密码（重复一次）"
+
+#: templates/auth/password_reset_confirm.html:21
+msgid "Change my password"
+msgstr "更改我的密码"
+
+#: templates/auth/password_reset_confirm.html:25
+msgid "Invalid password reset url."
+msgstr "无效的密码重置链接。"
+
+#: templates/auth/password_reset_done.html:8
+msgid ""
+"\n"
+"        We have sent an email to reset your password. Please check your "
+"inbox.\n"
+"    "
+msgstr ""
+"\n"
+"       我们已发送密码重置邮件。请查收您的 "
+"邮箱".\n"
+"    "
+
+#: templates/auth/password_reset_form.html:7
+msgid "Reset Password"
+msgstr "重置密码"
+
+#: templates/auth/password_reset_form.html:10
+#: templates/premises/new_premise.html:15
+msgid "Submit"
+msgstr "提交"
+
+#: templates/auth/profile.html:15
+msgid "update"
+msgstr "更新"
+
+#: templates/auth/profile.html:20
+msgid "Unfollow"
+msgstr "取消关注"
+
+#: templates/auth/profile.html:22
+msgid "Follow"
+msgstr "关注"
+
+#: templates/auth/profile.html:32 templates/search/profile.html:9
+#: templates/stats/profile.html:9
+msgid "FOLLOWER"
+msgstr "关注者"
+
+#: templates/auth/profile.html:36 templates/search/profile.html:13
+#: templates/stats/profile.html:13
+msgid "ARGUMENT"
+msgstr "命题"
```

Comment:
@tvvocold @plma @taiansu:
This issue might has something to do with Turkish-English translation, as Arguman.org was originally in Turkish.

The word "Premise" is "Önerme" on [the Turkish version of Arguman](http://tr.argument.org).
I can't read Turkish. According to Wikipedia, the translation of ["Önerme"](https://tr.wikipedia.org/wiki/%C3%96nerme) is ["Sentence"](https://en.wikipedia.org/wiki/Sentence_%28logic%29) in English, and ["句子"](https://zh.wikipedia.org/wiki/%E5%8F%A5%E5%AD%90_%28%E6%95%B0%E7%90%86%E9%80%BB%E8%BE%91%29) in Chinese.

Of course, using "句子" as the translation of "premise" still sounds weird, but it at least covers all three types of "premises" ("but", "because" and "however", they are all "句子".) 

I am _not_ suggesting that we should use "句子", but I think we should tend to use the conventional terms in Chinese, instead of being limited by the jargon in English.

@vargi  @fatiherikli: Could you please tell us about the standards you used when translating from Turkish to English? Did you tend to be strict with academic definitions, or just go with conventional expressions? Thanks!